source/extensions/filters/common/mutation_rules/mutation_rules.h

```diff
@@ -41,6 +41,7 @@ class Checker {
                     absl::string_view header_value) const;
 
 private:
+  bool canClearCache() const;
```

Comment:
This function seems not be used. Remove?

Reply:
Done.